### DISCUSSION TITLE
Imported Repository displays contributor statistics of only few contributors

### DISCUSSION THREAD

**Discussion by DishaDudhal** [community/community#64784] [Enterprise]
### Select Topic Area

Question

### Body

Preface: I am working on migrating a Gitlab Repo to Github Enterprise. In my Gitlab repo, there are 16 contributors with varied number of commits. However after migrating the repository (I basically cloned the repo to my local and the pushed it to Github remote) I can see only 4 of the contributors listed in the Contributors on Github. 

One big thing to consider is that not all the contributors on Gitlab are authorized to have the Github Enterprise account and license, and some of them have left the team too. But we still want to preserve the contributor-commit statistics (like the ones that are visible in Gitlab). 

A suggested approach to fix this was using `git filter-repo --mailmap email_map.txt`. The file email_map.txt will have mapping of username <old-email@oldcompany.com> <newemail@users.noreply.github.com> format on each row. Even doing this hasn't fixed the issue. 

Note that we cannot disburse new licenses to all these previous contributors but would still like to have them listed as contributors with their commit statistics in the Github repository. Can anyone suggest an approach to go about this?

Thanks!

**Comment by nirav-gajera** [comment]
The issue you are facing is that the GitHub Enterprise account and license is required to be listed as a contributor in the GitHub repository. However, you do not want to disburse new licenses to all the previous contributors.

One way to solve this issue is to create a new GitHub repository and copy the contents of the GitLab repository to the new GitHub repository. When you copy the contents of the GitLab repository, you can also copy the commit history. This will ensure that the contributor-commit statistics are preserved.

Once you have copied the contents of the GitLab repository to the new GitHub repository, you can then add the previous contributors as collaborators to the new GitHub repository. This will allow them to view the repository and their commit statistics.
